### PR TITLE
Feature/update

### PR DESCRIPTION
**Pull request by alfondotnet**

### CODE REVIEW

File: packages/eslint-config-quri-base/package.json [hunk 1]
```diff
@@ -44,17 +44,21 @@
...
+    "eslint-find-rules": "^1.14.0",
+    "eslint-plugin-import": "^2.0.1",
+    "eslint-plugin-react": "^6.5.0",
+    "in-publish": "^2.0.0",
+    "safe-publish-latest": "^1.1.1",
+    "tape": "^4.6.2"
   },
   "peerDependencies": {
-    "eslint": "^3.0.0",
-    "eslint-plugin-babel": "^3.3.0",
```
Comment (chollier) [inline packages/eslint-config-quri-base/package.json:57]: no more eslint-plugin-babel?

File: packages/eslint-config-quri-base/rules/best-practices.js [hunk 2]
```diff
@@ -63,161 +71,188 @@ module.exports = {
...
+
+    // disallow reassignments of native objects or read-only globals
+    // http://eslint.org/docs/rules/no-global-assign
+    'no-global-assign': ['error', { exceptions: [] }],
+    // deprecated in favor of no-global-assign
+    'no-native-reassign': 'off',
+
+    // disallow implicit type conversions
+    // http://eslint.org/docs/rules/no-implicit-coercion
+    'no-implicit-coercion': ['off', {
```
Comment (chollier) [inline packages/eslint-config-quri-base/rules/best-practices.js:106]: is this intended?

File: packages/eslint-config-quri-base/rules/imports.js [hunk 3]
```diff
@@ -15,44 +15,54 @@ module.exports = {
       node: {
         extensions: ['.js', '.json']
       }
-    }
+    },
+    'import/extensions': [
+      '.js',
+      '.jsx',
```
Comment (chollier) [inline packages/eslint-config-quri-base/rules/imports.js]: not sure we need jsx

File: packages/eslint-config-quri-base/rules/style.js [hunk 4]
```diff
@@ -67,78 +79,111 @@ module.exports = {
...
+    'max-depth': ['off', 4],
 
     // specify the maximum length of a line in your program
     // http://eslint.org/docs/rules/max-len
     'max-len': [1, 100, 2, {
       ignoreUrls: true,
-      ignoreComments: false
+      ignoreComments: false,
+      ignoreStrings: true,
+      ignoreTemplateLiterals: true,
```
Comment (chollier) [inline packages/eslint-config-quri-base/rules/style.js:133]: this is great

File: packages/eslint-config-quri-base/test/README.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
+# eslint-config-airbnb-base
```
Comment (chollier) [inline packages/eslint-config-quri-base/test/README.md]: need to rename here

File: packages/eslint-config-quri/package.json [hunk 6]
```diff
@@ -38,19 +38,24 @@
...
+    "eslint-plugin-jsx-a11y": "^2.2.3",
+    "eslint-plugin-react": "^6.4.1",
+    "in-publish": "^2.0.0",
+    "react": ">= 0.13.0",
+    "safe-publish-latest": "^1.1.1",
+    "tape": "^4.6.2"
+    },
+    "peerDependencies": {
+    "eslint": "^3.8.1",
+    "eslint-plugin-jsx-a11y": "^2.2.3",
```
Comment (chollier) [inline packages/eslint-config-quri/package.json]: we don't need this one

File: packages/eslint-config-quri/rules/react.js [hunk 7]
```diff
@@ -218,29 +235,78 @@ module.exports = {
...
-    // disallow target="_blank" on links
+    // Disallow target="_blank" on links
     // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md
-    // TODO: enable
-    'react/jsx-no-target-blank': 0
+    'react/jsx-no-target-blank': 'error',
+
+    // only .jsx files may have JSX
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-filename-extension.md
+    'react/jsx-filename-extension': ['error', { extensions: ['.jsx'] }],
```
Comment (chollier) [inline packages/eslint-config-quri/rules/react.js]: hmm?

**Review by chollier — COMMENTED** [review]
I like that they change 0 / 1 / 2 to 'off' / 'error' / 'never' 

left a few comments, thanks for doing this 🙇

**Comment by alfondotnet** [comment]
Thanks a lot for the comments, after a while I got a little crazy but later in the day I will fix these

File: packages/eslint-config-quri/rules/react.js (hunk 7 above)
Comment (alfondotnet) [reply packages/eslint-config-quri/rules/react.js]: :+1:

File: packages/eslint-config-quri/package.json (hunk 6 above)
Comment (alfondotnet) [reply packages/eslint-config-quri/package.json]: 👍

File: packages/eslint-config-quri/package.json [hunk 8]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-quri",
-  "version": "3.0.1",
+  "version": "3.0.2",
```
Comment (chollier) [inline packages/eslint-config-quri/package.json]: I would do a major bump here

File: packages/eslint-config-quri/test/package.json [hunk 9]
```diff
@@ -41,14 +41,14 @@
...
   },
-  "homepage": "https://github.com/airbnb/javascript",
+  "homepage": "https://github.com/quri/javascript",
   "dependencies": {
-    "eslint-config-airbnb-base": "^9.0.0"
+    "eslint-config-quri-base": "^9.0.0"
   },
   "devDependencies": {
-    "babel-preset-airbnb": "^2.1.1",
+    "babel-preset-quri": "^2.1.1",
```
Comment (chollier) [inline packages/eslint-config-quri/test/package.json]: we don't have that package

**Pull request merged**